### ISSUE TITLE
Chart to image how-to

### ISSUE THREAD

**Issue by giraone** [chartist-js/chartist#862]
I was looking for a way to export chartist.js charts to an image or PDF. There were a lot of questions on stackoverflow and also here in the issues list, but I found no real solution. So I invested some time and created a documentation, how to do this on the client side. The result is documented in [https://github.com/giraone/chartist2image](https://github.com/giraone/chartist2image). The same problems I detected will also occur, if one will try the same on the server side, because the 2 main problems `<foreignObject>` and external CSS styles are general problems of SVG to image conversions. 

Basically I only want to share this information and maybe you can think about offering an option for suppressing the usage of <foreignObject>, because at the moment my solution may not work anymore, if you change the internal `supportsForeignObject` code.

**Comment by alexgig** [comment]
Hi @giraone,

You may find [node-chartist](https://github.com/panosoft/node-chartist) useful as it addresses these issues and allows you to create static Chartist charts on the server (or on the client for that matter if you browserify the source).

The HTML/SVG output can then be passed to something like [PrinceXML](https://www.princexml.com/) to generate a pdf or [wkhtmltoimage](http://wkhtmltopdf.org/) to create a png.

Hope that helps!

Best,
Alex

**Comment by deQuota** [comment]
Hi, I had the same problem to convert chartist charts to images. I was able to do it and I have created a npm module for that,
hope now the problem will solved 

https://www.npmjs.com/package/chartist-to-image
